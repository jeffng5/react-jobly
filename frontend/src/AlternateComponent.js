import React, {useState, useEffect, useContext} from 'react'

//Alternate component is used to display applied jobs 

const AlternateComponent = ({id, title}) => {
    

    return (
    <>

    
      <div className="card-body">
        <h6 className="card-title">
    
          <h4>{id}</h4>
          <h2>You applied for: {title}</h2>

         
        </h6>
       
      </div>

    </>)


    }


    export default AlternateComponent