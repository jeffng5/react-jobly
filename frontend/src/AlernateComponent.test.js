import React from 'react';
import { render } from '@testing-library/react';
import AlternateComponent from './AlternateComponent'

it('renders without crashing', function() {
  render(<AlternateComponent />);
})

//snapshot test
it("matches snapshot", function() {
    const {asFragment} = render(<AlternateComponent />);
    expect(asFragment()).toMatchSnapshot();
  });

test('testing component', ()=> {
    const {getByText} = render( <AlternateComponent />);
    getByText('applied')
})