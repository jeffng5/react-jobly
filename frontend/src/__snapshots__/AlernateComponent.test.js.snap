// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="card-body"
  >
    <h6
      class="card-title"
    />
    <h4 />
    <h2>
      You applied for: 
    </h2>
  </div>
</DocumentFragment>
`;
